Makefile

```diff
@@ -1,26 +1,12 @@
-HERE = $(shell pwd)
-VENV = .
-VIRTUALENV = virtualenv
-BIN = $(VENV)/bin
-PYTHON = $(BIN)/python
+.PHONY: build run
 
-INSTALL = $(BIN)/pip install --no-deps
+SERVICEBOOK ?= http://192.168.1.12:5000/api/
```

Comment:
where this ip comes from?

Reply:
No idea. It's from https://github.com/mozilla/servicebook-web/blob/master/docker/Makefile#L3 😝

Reply:
(by that I mean I didn't introduce it)